cpp/src/log.cpp

```diff
@@ -26,7 +26,7 @@ void parseLogLevel()
       });
 
     auto level = logLevelNames.find(logLevelName);
-    if (level != logLevelNames.end())
+    if (!logLevelName.empty() > 0 && level != logLevelNames.end())
```

Comment:
`logLevelName.empty()` returns bool, why the check `> 0` ?

Is this just:
```
if (!logLevelName.empty() && level != logLevelNames.end())
```
?

Reply:
Yes, thanks for catching, fixed in https://github.com/rapidsai/ucxx/pull/50/commits/a811406d5af37375c0d64c06f5c0dc80e0a81644. I remember when I was writing this I was doing it the Python way, `logLevelName.size() > 0`, then I remembered of `.empty()` but forgot to remove the comparison operator.